app/src/main/java/org/p2p/wallet/infrastructure/network/environment/NetworkServicesUrlProvider.kt

```diff
@@ -50,11 +49,11 @@ class NetworkServicesUrlProvider(
         ).orEmpty()
         val verifier = sharedPreferences.getString(
             KEY_TORUS_BASE_VERIFIER,
-            context.getString(R.string.torusFeatureVerifier)
+            context.getString(R.string.torusReleaseVerifier)
         ).orEmpty()
         val subVerifier = sharedPreferences.getString(
             KEY_TORUS_BASE_SUB_VERIFIER,
-            emptyString()
+            context.getString(R.string.torusReleaseSubVerifier)
```

Comment:
Maybe we should pass here the certain data according to the build as well?
If DEBUG it's empty, if not, we are passing release verifier mb?

Reply:
we can
but in debug we can change all this data from debug menu